def error1():
    print("error")